translations/bnpchs-final-2018-super-trombosit.html

```diff
@@ -0,0 +1,109 @@
+<!-- Title : Super Thrombocyte -->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  There is new research in immunology, called Super Thrombocyte. 
+  A Super Thrombocyte is able to heal injuries in the body much faster compared to a normal thrombocyte. 
+  According to research, the number of Super Thrombocyte required to heal an injury is 
+  dependent on three variables: $$$X$$$, $$$Y$$$, and $$$Z$$$. These variables are all integers.
+</p>
+
+<p style="text-align: justify;">
+  The number of Super Thrombocyte that is required follows the following formula:
+</p>
+
+<p style="text-align: justify;">
+  $$${\Large \textrm{ST}(X,Y,Z) =  \textrm{GCD}(X^1 + Y, X^2 + Y, X^4 + Y, \dots, X^{2^Z} + Y)}$$$
+</p>
+
+
+<p style="text-align: justify;">
+  Currently, you have three arrays: $$$P$$$, $$$Q$$$, and $$$R$$$, each containing $$$N$$$ integers. 
+  Array $$$P$$$ stores all possible values of $$$X$$$. 
+  Array $$$Q$$$ stores all possible values of $$$Y$$$. 
+  Array $$$R$$$ stores all possible values of $$$Z$$$.
+</p>
+
+<p style="text-align: justify;">
+  Your task is to calculate the total number of <b>Super Thrombocytes required for all possible combinations of values $$$X$$$, $$$Y$$$, and $$$Z$$$</b>. 
```

Comment:
```suggestion
  Your task is to calculate the <b>total number of Super Thrombocytes required for all possible combinations of values $$$X$$$, $$$Y$$$, and $$$Z$$$</b>. 
```
bold-nya dari "jumlah Super Trombosit ..."